README.md

```diff
@@ -1,12 +1,12 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.14283584.svg)](https://doi.org/10.5281/zenodo.14283533)
-![PyPI - Version](https://img.shields.io/pypi/v/streetscapes)
+[![PyPI - Version](https://img.shields.io/pypi/v/streetscapes)](https://pypi.org/project/streetscapes/)
 [![Research Software Directory](https://img.shields.io/badge/RSD-streetscapes-00a3e3)
 ](https://research-software-directory.org/software/streetscapes)
 
-# `streetscapes`
+# Introduction
 This repository contains information and code for downloading, segmenting and analysing images from Mapillary and KartaView, using information from [global-streetscapes](https://github.com/ualsg/global-streetscapes/tree/main) dataset.
 
-# Installation
+# Setup
```

Comment:
I don't know about this, it's going into user micromanagement territory. I would say it's pretty obvious that they need to clone the repository since the notebooks use functions from the package.

Reply:
Perhaps we can discuss in person. You can use the library by installing from pypi, so no need to clone the repo.

Reply:
Yes, sorry - I meant that they have to ***install*** the library at any rate, regardless of the method. But I see your point, I expanded this section and added a note about notebooks.